aspnetcore/blazor/security/includes/httpcontext.md

```diff
@@ -1,5 +1,38 @@
+:::moniker range=">= aspnetcore-8.0"
+
+<xref:Microsoft.AspNetCore.Http.IHttpContextAccessor> must be avoided when interactive rendering for the following reasons:
+
+* The interface isn't implemented for interactive WebAssembly rendering.
+* For interactive server rendering, the implementation isn't guaranteed to exist.
+
+<xref:Microsoft.AspNetCore.Http.IHttpContextAccessor> can be used for components that are statically rendered on the server. **However, we recommend avoid using it if possible.**
```

Comment:
```suggestion
<xref:Microsoft.AspNetCore.Http.IHttpContextAccessor> can be used for components that are statically rendered on the server. **However, we recommend to avoid using it if possible.**
```